packages/components/nodes/vectorstores/Vectara_Existing/Vectara_Existing.ts

```diff
@@ -34,7 +34,7 @@ class VectaraExisting_VectorStores implements INode {
             {
                 label: 'Vectara Metadata Filter',
                 name: 'filter',
-                type: 'json',
+                type: 'string',
                 additionalParams: true,
```

Comment:
can we add a `description` with how to use the filter, or the link to the docs page? Something like:

```
Refer to <a target="_blank" href="https://docs.vectara.com">official guide</a> on how to use filter
```

Reply:
This has been added in https://github.com/FlowiseAI/FlowiseDocs/pull/22

Reply:
thanks for the docs PR! would be good to add a description property as well

`description: 'Refer to....'`

Reply:
Ah I didn't know there was a description property. Will add that as well!

Reply:
@HenryHengZJ Added the description property. Let me know if there is anything else I should change.